react/components/Footer/components/FooterPoweredBy.js

```diff
@@ -0,0 +1,20 @@
+import React, { PureComponent } from 'react'
+
+import VTEXIcon from './../images/VTEX-BW.svg'
+
+class FooterPoweredBy extends PureComponent {
+  static displayName = 'FooterPoweredBy'
+  static contextTypes = {
```

Comment:
Use the `withRuntimeContext` hoc from `render` and grab the account from the props, instead of the old context API.

Reply:
@lucasecdb Alterado.